codes/config.py

```
@@ -38,3 +38,32 @@ def __getitem__(self, key: str):
     def epsilon(self) -> float:
         # Alias to maintain old API: return initial epsilon value
         return self.epsilon_start
+    def __post_init__(self):
+        # Trust-related params
+        if not (0.0 <= self.initial_trust <= 1.0):
+            raise ValueError(f"initial_trust must be in [0.0, 1.0], got {self.initial_trust}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)